docs/installing-packages.md

```diff
@@ -0,0 +1,203 @@
+    * [Install packages through 'packageList'](#install-packages-through-packagelist)
+        * [Install a package from a third-party repo](#install-a-package-from-a-third-party-repo)
+        * [Install a package from SUSE's internal repositories](#install-a-package-from-suses-internal-repositories)
+    * [Side-load RPMs](#side-load-rpms)
+        * [RPM with dependency resolution from a third-party repository](#rpm-with-dependency-resolution-from-a-third-party-repository)
+        * [RPM with depdendency resolution from SUSE's internal repositories](#rpm-with-depdendency-resolution-from-suses-internal-repositories)
+    * [Installing unsigned packages](#installing-unsigned-packages)
+3. [Package installation workflow](#package-installation-workflow)
+    * [Running the EIB container](#running-the-eib-container)
+    * [Building the EIB image](#building-the-eib-image)
+        * [RPM resolution process](#rpm-resolution-process)    
+        * [Troubleshooting](#troubleshooting)
+    * [Booting the EIB image](#booting-the-eib-image)
+
+## Supported systems
+EIB's **RPM resolution** process and package installation has been tested on the following `x86_64` systems: 
+1. [Ubuntu 22.04](https://releases.ubuntu.com/jammy/)
+2. [SLES 15-SP5](https://www.suse.com/download/sles/)
+3. [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed/)
+4. [Fedora Linux](https://fedoraproject.org/server/download)
+
+## Configure packages for installation
+You can configure packages for installation in the following ways:
+1. provide a `packageList` configuration under `operatingSystem.packages` in the EIB image configuration file
+2. create a `rpms` directory under EIB's configuration directory and provide local RPM files that you want to be installed on the image
+
+### Install packages through 'packageList'
+To install a package using the `packageList` configuration, at a minimum you must configure the following under `operatingSystem.packages`:
+1. valid package names under `packageList`
+2. `additionalRepos` or `sccRegistrationCode`
+
+Below you can find examples for the aforementioned use cases.
+
+#### Install a package from a third-party repo
+```yaml
+operatingSystem:
+  packages:
+    packageList:
+      - reiserfs-kmp-default-debuginfo
+    additionalRepos:
+      - url: https://download.opensuse.org/repositories/Kernel:/SLE15-SP5/pool
+```
+> **_NOTE:_** Before adding any repositories under `additionalRepos`, make sure that they are signed with a valid GPG key. **All non-signed additional repositories will cause EIB to fail.**
+
+#### Install a package from SUSE's internal repositories
+```yaml
+operatingSystem:
+  packages:
+    packageList:
+      - wget2
+    sccRegistrationCode: <your-reg-code>
+```
+
+### Side-load RPMs
+Sometimes you may want to install RPM files that are local to your machine. For this use-case, you should create a directory called `rpms` under `<eib-config-dir>/rpms` and copy your local RPM files there.
```

Comment:
Nit: Consider rephrasing this as "you may want to install RPM files that are not hosted in a repository." That's a bit broader of a rationale than simply having the file on your machine.